packages/wonder-blocks-spacing/package.json

```diff
@@ -1,19 +1,22 @@
 {
   "private": true,
-  "name": "wonder-blocks-spacing",
+  "name": "@khanacademy/wonder-blocks-spacing",
   "version": "0.0.1",
   "design": "v1",
+  "publishConfig": {
```

Comment:
🤷‍♂️ What does it mean when a package has a public publishConfig but is marked as `"private": true`?

Reply:
Yeah... the terminology is a little wonky. `"private": true` prevents publishing completely, `"publishConfig"` sets specific settings at publish time only, in this case the "access" setting does the following:
> When publishing scoped packages, the access level defaults to restricted. If you want your scoped package to be publicly viewable (and installable) set --access=public. The only valid values for access are public and restricted. Unscoped packages always have an access level of public.